<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Hitesh-Mac-Mini.local" name="com.eduven.testcase.ScreenCaptureTestCases" tests="3" failures="0" timestamp="12 Aug 2016 09:40:32 GMT" time="997.946" errors="1">
  <testcase name="inAppPurchaseTest" time="286.758" classname="com.eduven.testcase.ScreenCaptureTestCases"/>
  <testcase name="createDirectoriesTest" time="163.568" classname="com.eduven.testcase.ScreenCaptureTestCases"/>
  <testcase name="screenShotCaptureForCategoryPage" time="547.620" classname="com.eduven.testcase.ScreenCaptureTestCases">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.eduven.modules.AutoPlay.clickOnPlayButton(AutoPlay.java:204)
	at com.eduven.testcase.ScreenCaptureTestCases.screenShotCaptureForCategoryPage(ScreenCaptureTestCases.java:98)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
    </error>
  </testcase> <!-- screenShotCaptureForCategoryPage -->
</testsuite> <!-- com.eduven.testcase.ScreenCaptureTestCases -->
